deployments/charts/kuma/values.yaml

```diff
@@ -20,6 +20,10 @@ installCrdsOnUpgrade:
 # -- Whether to disable all helm hooks
 noHelmHooks: false
 
+# -- Namespaces that are part of the Mesh. Required to create RoleBindings in these namespaces.
+# If not specified, a ClusterRoleBinding will be created instead.
```

Comment:
Could you refine this comment a bit? It would be more helpful to focus on what the outcome will be when this is used, rather than explaining why it's required or not